diffpy/snmf/new_subroutines.py

```diff
@@ -6,8 +6,22 @@
 
 
 def create_components(number_of_components, grid_vector, number_of_signals, signal_length):
+    """Creates the ComponentSignal objects
+
+    Parameters
+    ----------
+    number_of_components: int
```

Comment:
the docstring should say what they are:
`the number of component signals in the NMF decomposition`
etc.

Reply:
Ok. I will change them